k8s/cli.py

```diff
@@ -112,6 +113,19 @@ class Default(Enum):
             "help": "Look only within this namespace",
         },
     ),
+    (
+        "--ignore",
+        {
+            "dest": "expressions",
+            "action": "store",
+            "type": str,
+            "nargs": "?",
+            "default": Default.ignore.value,
+            "required": False,
+            "help": "Comma delimited list of expressons to match against \
```

Comment:
I think it would be better to specify multiple `--ignore` flags, instead of doing a comma separated list. It's both easier to use when scripting, and I think it's also easier to parse. It also has the benefit of allowing customers to ignore names with commas in them.

Reply:
Understood. However, --namespace also uses comma separated list. Should we make these two consistent? or just let them have different methods.